docs/examples/evaluation/prometheus_evaluation.ipynb

```diff
@@ -0,0 +1,1498 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    gpt4_llm = OpenAI("gpt-4")
</pre><p>default temperature = 0.1, but we set temperature for prometheus to 0.7. I know its hard to get apples-to-apples with this value for different models, but perhaps 0.7 temp is leading to greater amount of hallucinations.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9406/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Yeah. I will rerun and update.

Reply:
Did not change the results as discussed. It should be because of quantization.